src/portal/src/lib/components/config/gc/gc-history/gc-history.component.html

```diff
@@ -5,13 +5,15 @@ <h5 class="history-header" id="history-header">{{'GC.JOB_HISTORY' | translate}}<
 <clr-datagrid [clrDgLoading]="loading">
   <clr-dg-column>{{'GC.JOB_ID' | translate}}</clr-dg-column>
   <clr-dg-column>{{'GC.TRIGGER_TYPE' | translate}}</clr-dg-column>
+  <clr-dg-column>{{'TAG_RETENTION.DRY_RUN' | translate}}</clr-dg-column>
   <clr-dg-column>{{'STATUS' | translate}}</clr-dg-column>
   <clr-dg-column>{{'CREATION_TIME' | translate}}</clr-dg-column>
   <clr-dg-column>{{'UPDATE_TIME' | translate}}</clr-dg-column>
   <clr-dg-column>{{'LOGS' | translate}}</clr-dg-column>
   <clr-dg-row *ngFor="let job of jobs" [clrDgItem]='job'>
     <clr-dg-cell>{{job.id }}</clr-dg-cell>
     <clr-dg-cell>{{(job.type ? 'SCHEDULE.'+ job.type.toUpperCase() : '') | translate }}</clr-dg-cell>
+    <clr-dg-cell>{{isDryRun(job?.parameters)}}</clr-dg-cell>
```

Comment:
I think we can use 'Yes/No' to replace 'true/false'

Reply:
make sence. modified